v19.1/performance-benchmarking-with-tpc-c.md

```diff
@@ -261,7 +261,7 @@ CockroachDB offers a pre-built `workload` binary for Linux that includes several
     ~~~ shell
     $  ./workload.LATEST fixtures load tpcc \
     --warehouses=10000 \
-    "postgres://root@<node1 address>?sslmode=disable postgres://root@<node2 address>?sslmode=disable postgres://root@<node3 address>?sslmode=disable [...space separated list]"
+    "postgres://root@<node1 address>?sslmode=disable"
```

Comment:
Don't you still want to include all the nodes in the tpcc command line?

Reply:
Based on the info in #4287, the load command should point to a single node, not a list. Am I misunderstanding?

Reply:
Ah, got it. `fixtures load` just wants one URL but the `run` part uses more than one.